src/01_bignum.py

```diff
@@ -1,4 +1,9 @@
 # Print out 2 to the 65536 power
 # (try doing the same thing in the JS console and see what it outputs)
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+
+x = 2
+y = 65536
+
+x ** y
```

Comment:
I think the idea here is to print this to get an idea of how massive that number is :)